v23.1/create-schedule-for-changefeed.md

```diff
@@ -0,0 +1,154 @@
+
+You can include the changefeed options listed on the [`CREATE CHANGEFEED`](create-changefeed.html#options) page to modify the behavior of your changefeed. The following options are **not** compatible with scheduled changefeeds:
+
+- `diff`
+- `end_time`
+- `mvcc_timestamp`
+- `resolved`
+- `updated`
+
+Scheduled changefeeds have the [`initial_scan = 'only'`](create-changefeed.html#initial-scan) option included implicitly. You cannot specify `initial_scan` as `'yes'` or `'no'`.
+
+## Schedule options
+
+Option | Value | Description
+-------+-------+-------------
+`first_run` | [`TIMESTAMP`](timestamp.html) / `now` | Execute the first run of the schedule at this time. If you do not specify `first_run`, the schedule will execute based on the next `RECURRING` time set by the crontab. 
+`on_execution_failure` | `retry` / `reschedule` / `pause` | Determine how the schedule handles an error. <br><br>`retry`: Retry the changefeed immediately.<br><br>`reschedule`: Reschedule the changefeed based on the `RECURRING` expression.<br><br>`pause`: Pause the schedule. This requires that you [resume the schedule](resume-schedules.html) manually.<br><br>**Default:** `reschedule`
+`on_previous_running` | `start` / `skip` / `wait` | Control whether the changefeed schedule should start a changefeed if the previous scheduled changefeed is still running.<br><br>`start`: Start the new changefeed anyway, even if the previous one is running.<br><br>`skip`: Skip the new changefeed and run the next changefeed based on the `RECURRING` expression.<br><br>`wait`: Wait for the previous changefeed to complete.<br><br>**Default:** `wait`
+
+## Examples
+
+Before running any of the examples in this section, it is necessary to enable the `kv.rangefeed.enabled` cluster setting. If you are working on a {{ site.data.products.serverless }} cluster, this cluster setting is enabled by default.
+
+The [Changefeed Sinks](changefeed-sinks.html) page provides detail on the available sinks for your change data messages and connection URIs. We recommend using [external connections](create-external-connection.html) to interact with external sinks. The examples in this section use an external connection URI for the changefeed sink.
+
+### Create a scheduled changefeed
+
+The following statement sets up a scheduled changefeed named `users_rides_nightly` that will send changefeed messages in `CSV` format 1 minute past midnight every night. As soon as the statement is run, the first changefeed run will execute immediately:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE SCHEDULE users_rides_nightly FOR CHANGEFEED INTO 'external://kafka-sink' WITH format=csv RECURRING '1 0 * * *' WITH SCHEDULE OPTIONS first_run=now, on_execution_failure=retry, on_previous_running=skip;
+~~~
+
+If the scheduled changefeed runs into an error, it will retry the changefeed immediately. The changefeed will skip a new changefeed run if the previous run is still running at the next scheduled time.
```

Comment:
nit: I really like the bulleted breakdown of what this command does that you've used on `v23.1/export-data-with-changefeeds.md` lines 47-50. Could that be replicated here for clarity? Notably, the pairing of, say, "`on_previous_running=skip` means X will happen" helps to parse the whole command at a glance. Optional!

Reply:
Done, good call!